### ISSUE TITLE
How and where to get these datasets?

### ISSUE THREAD

**Issue by ThanatosXingYu** [VisionRush/DeepFakeDefenders#5]
I'm a newbie in AI, so I wander to know how can I get the datasets and how to change the hyper-parameters

I don't know the difference between `dataset/train.txt` and `dataset/val.txt`

**Comment by XinyiZhangcasia** [comment]
Hi, I'm glad to see you're interested in deepfake topics! Currently, the dataset is not yet open-sourced, the original datasets can be found in the official page of this competition.

As for the hyper-parameters, most of them are listed at the beginning of the `main_train.py` . However, a few specific ones, such as the learning rate strategy and model definitions, can be found in `core/mengine.py` and `toolkit/chelper.py`. You can modify them as needed in the respective locations.

Regarding the `dataset/train.txt` and `dataset/val.txt`, they are just examples. You'll need to replace them with the paths to your own data, formatted similarly to the examples, for training and validation purposes.

**Comment by XinyiZhangcasia** [comment]
Here is the official link. [https://www.kaggle.com/competitions/multi-ffdi/data](url)

Of course you can change hyper-parameters in `main_train.py`.

Before running the training, please make sure to download the ImageNet-1K pre-trained weights for the two models from the following links and place them in the `./pre_model` directory:

RepLKNet: [https://drive.google.com/f ile/d/1vo-P3XB6mRLUeDzmgv90dOu73uCeLfZN/view?usp=sharing](url)
ConvNeXt: [https://dl.fbaipublicfiles.com/convnext/convnext base 1k 384.pth](url)

**Comment by ThanatosXingYu** [comment]
OK, thanks!

**Comment by XksA-me** [comment]
> > https://drive.google.com/f ile/d/1vo-P3XB6mRLUeDzmgv90dOu73uCeLfZN/view?usp=sharing
> 
> ! [image] (https://github.com/user-attachments/assets/707ce6df-5ac7-4914-b337-854d8f8a62c9) This address is invalid and cannot be opened, is there a public model?

```bash
RepLKNet: https://drive.google.com/file/d/1vo-P3XB6mRLUeDzmgv90dOu73uCeLfZN/view?usp=sharing
ConvNeXt: https://dl.fbaipublicfiles.com/convnext/convnext_base_1k_384.pth
```
this is ok.
